docs/install/server.md

```diff
@@ -111,6 +111,29 @@ The possible flags are:
     file.</p>
 </div>
 
+### Install via Homebrew (Mac OS X)
+
+You can use the Homebrew package manager to install PostgREST on Mac
+
+```bash
+# Ensure brew is up to date
+brew update
+
+# Check for any problems with brew's setup
+brew doctor
+
+# Install the postgrest package
+brew install postgrest
+```
+
+This will automatically install PostgreSQL as a dependency (see the [Installing PostgreSQL](#installing-postgresql) section for setup instructions). The process tends to take around 15 minutes to install the package and its dependencies.
```

Comment:
Probably want to say "up to 15 minutes" as if they are installed into `/usr/local` they'll get binary packages which just take download + extraction time.